app/views/home/index.html.erb

```diff
@@ -54,21 +54,15 @@
                                              theme: :accent,
                                              classes: 'rounded-xl p-4 mb-4',
                                              show_arrows: :side) do |carousel| %>
-      <%# Until we  have more things to go in the carousel, do this for demo purposes %>
-      <% %w[funnel-pupils.png funnel-teachers.png].each do |pic| %>
-        <%= carousel.with_grid(cols: 2, cell_classes: 'my-auto') do |grid| %>
-          <%= grid.with_image(src: pic, classes: 'rounded-xl fit') %>
-          <%= grid.with_testimonial_card do |testimonial| %>
-            <%= testimonial.with_header(title: t('home.testimonials.card_1.title')) %>
-            <%= testimonial.with_quote { "#{t('home.testimonials.card_1.quote')}." } %>
-            <%= testimonial.with_source(classes: 'text-blue-very-dark') do %>
-              <strong><%= t('home.testimonials.card_1.source.name') %></strong><br>
-                <%= t('home.testimonials.card_1.source.school') %>
-            <% end %>
-            <%= testimonial.with_button(t('home.testimonials.read_case_study'), 'case-studies/15', style: :primary) %>
-            <%= testimonial.with_button(t('home.testimonials.more_case_studies'), case_studies_path,
-                                        outline_style: :transparent) %>
-          <% end %>
+      <% Testimonial.all.active.default.each do |testimonial| %>
```

Comment:
Maybe add `.shuffle` to randomise the order a bit?

Reply:
Good idea 👍🏻